content/en/docs/installation/deployment-reg.md

```diff
@@ -12,23 +12,25 @@ toc_hide: true
 
 ## Overview
 
-As part of the configuration process, you need to register your instance with Armory.
+As part of the configuration process, you need to register your Armory Enterprise instance. If you maintain several instances, such as development, staging, and production instances, you must register each one.
 
 ## Register your environment
 
 1. Navigate to the URL provided by Armory and follow the instructions to create an account.
 2. Make note of the following information:
-  - Instance ID
-  - Client ID
-  - Secret
-   > This is the only time you are shown the secret value. Store it somewhere safe.
+
+   - Instance ID
+   - Client ID
+   - Secret
+   
+   **This is the only time you are shown the secret value. Store it somewhere safe.**
 
 ## Apply the registration information
 
 In your Operator manifest (such as `spinnakerService.yml`) or `spinnaker-local` file (Halyard), configure the following parameters:
   - `spec.spinnakerConfig.profiles.spinnaker.”armory.cloud”.iam.tokenIssueUrl`: set this value to `https://auth.cloud.armory.io/oauth/token`.
-  - `spec.spinnakerConfig.profiles.spinnaker.”armory.cloud”.iam.clientID`: set this value to **Client ID** that you received from [Get your registration information](#get-your-registration-information)
-  - `spec.spinnakerConfig.profiles.spinnaker.”armory.cloud”.iam.clientSecret`: set this value to the **Secret** that you received from [Get your registration information](#get-your-registration-information).
+  - `spec.spinnakerConfig.profiles.spinnaker.”armory.cloud”.iam.clientID`: set this value to **Client ID** that you received from [Register your environment](#register-your-environment).
```

Comment:
AFAIK, the armory.cloud bit doesn't need to be in quotes. The dots are stand-ins for indentation in yaml. 

This:
```
armory.cloud:
  iam: ...
```
is the same as this:
```
armory:
  cloud:
    iam: ...
```